smart-contracts/osmosis/contracts/cl-vault/src/vault/swap.rs

```diff
@@ -34,7 +34,7 @@ pub struct SwapParams {
 
 pub fn execute_swap_non_vault_funds(
     deps: DepsMut,
-    env: Env,
+    contract_address: String,
```

Comment:
Changing the type to `Addr` improves type safety

Reply:
done